src/phases/filesystem/FlushingTemplate.yml

```diff
@@ -0,0 +1,1656 @@
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+      - *defaultDocument
+
+Actors:
+  #  # Phase 0: insert.
+  - Name: Insert
+    Type: CrudActor
+    Database: {^Parameter: {Name: "Database", Default: *database}}
+    Threads: {^Parameter: {Name: "Threads", Default: *insertThreads}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: [0]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Repeat: {^Parameter: {Name: "Iterations", Default: *iterationsCount}}
+          Database: {^Parameter: {Name: "Database", Default: *database}}
```

Comment:
Oh, you are not consistent in your use of ^Parameter between these settings and some others. Is there a reason to not set these at the top like the others?